packages/backend/src/app-connection/app-connection-entity.ts

```diff
@@ -0,0 +1,46 @@
+import { AppConnection, Project } from "shared";
+import { EntitySchema } from "typeorm";
+import { ApIdSchema, BaseColumnSchemaPart } from "../helper/base-entity";
+
+export type AppConnectionSchema = AppConnection & { project: Project };
+
+export const AppConnectionEntity = new EntitySchema<AppConnectionSchema>({
+  name: "app_connection",
+  columns: {
+    ...BaseColumnSchemaPart,
+    name: {
+      type: String,
+    },
+    appName: {
+      type: String
+    },
+    projectId: ApIdSchema,
+    value: {
+      type: "jsonb"
+    }
+  },
+  indices: [
+    {
+      name: "idx_app_connection_project_id_and_app_name_and_name",
+      columns: ["projectId", "appName", "name"],
+      unique: true,
+    },
+    {
+      name: "idx_app_connection_project_id_and_name",
+      columns: ["projectId", "name"],
+      unique: true,
+    },
```

Comment:
seems like the unique constraint on `["projectId", "appName", "name"]` is redundant.